kinetic/distribution.yaml

```diff
@@ -9514,6 +9514,16 @@ repositories:
       url: https://github.com/appliedAI-Initiative/orb_slam_2_ros.git
       version: master
     status: maintained
+  orb_slam2_with_maps-odom:
```

Comment:
From the [reviewing guidelines](https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md):

> The source repository should generally contain one or more ROS packages (ROS packages are ones that have a package.xml and use either catkin or ament build helpers). Packages that are not ROS packages can be accepted, but they should be rare and require special handling in the release repository.

I don't think that the repository satisfies that requirement, as there aren't any ros packages in it.

Reply:
Yeah, without ROS packages in it we cannot index it or generate documentation. I'm going to close this.

Reply:
I have changed repository.